### PR TITLE
Add syntactic sugar to reach the current folder.

### PR DESCRIPTION
**Pull request by yageek**
Add the possibility to reach the current folder with:
```swift
import Files

var current = Folder.current
```
This is an alternative to `FileSystem().currentFolder`.

We could imagine add the same for the home folder, temporary folder and so on.

### CODE REVIEW

File: Sources/Files.swift [hunk 1]
```diff
@@ -485,6 +485,11 @@ public final class Folder: FileSystem.Item, FileSystemIterable {
     public var subfolders: FileSystemSequence<Folder> {
         return makeSubfolderSequence()
     }
+
+    /// A reference to the folder that is the current working directory.
+    public static var current: Folder {
+        return try! Folder(path: "")
```
Comment (JohnSundell) [inline Sources/Files.swift]: Could you make this return `FileSystem(fileManager: fileManager).currentFolder` instead? That way, any custom file manager is preserved, and we don't have to introduce another force unwrapping 🙂

**Comment by yageek** [comment]
I updated the PR. I have added the `home` and `temporary` static methods too.

File: Sources/Files.swift [hunk 2]
```diff
@@ -485,6 +485,21 @@ public final class Folder: FileSystem.Item, FileSystemIterable {
     public var subfolders: FileSystemSequence<Folder> {
         return makeSubfolderSequence()
     }
+
+    /// A reference to the folder that is the current working directory
+    public static var current: Folder {
+        return FileSystem(using: .default).currentFolder
```
Comment (JohnSundell) [inline Sources/Files.swift]: Don't use `.default`, use the Folder's `fileManager`. That way, if a custom one is used, it's not lost.

Comment (yageek) [inline Sources/Files.swift]: How can I achieve that as `current` is a static variable ?

Comment (JohnSundell) [inline Sources/Files.swift]: Ah, sorry, shouldn't do code reviews when I'm this tired! 😅 You are completely right 👍

**Pull request merged**